.github/workflows/test.yml

```diff
@@ -146,5 +146,6 @@ jobs:
           export LD_LIBRARY_PATH=${R_HOME}/lib
           cargo test --manifest-path extendr-engine/Cargo.toml --features libR-sys/use-bindgen -- --nocapture --test-threads=1
           cargo test --manifest-path extendr-api/Cargo.toml --features libR-sys/use-bindgen -- --nocapture --test-threads=1
+          cargo test --manifest-path extendr-api/Cargo.toml --features libR-sys/use-bindgen --features ndarray -- --nocapture --test-threads=1
```

Comment:
I've been wondering whether we should add a feature `tests-all` and add `ndarray` and `libR-sys/use-bindgen` (if possible, not sure if it works for dependent libs) to that feature. Then the test code never needs to change as we add new features, and we also don't get a super long argument as we add a larger set of features.

Reply:
To be clear: I would add the `tests-all` feature to all three crates, so the tests for extendr-api, extendr-engine, etc look all the same, even if those crates have different actual features they offer.

Reply:
Thanks for the idea, I added `tests-all`. I found we also need another flavor for testing without ndarray. I'm not sure if this is more handy than listing individual features. What do you think?

Reply:
See my other comment.